src/client/components/UserPage/UserLinkTable/ToolBar/FilterSortPanel/widgets/CheckIcon.tsx

```diff
@@ -1,24 +1,22 @@
 import React, { FunctionComponent } from 'react'
 
 type CheckIconProps = {
-  size: number
   color?: string
   className?: string
 }
 
 const CheckIcon: FunctionComponent<CheckIconProps> = ({
-  size,
   color = '#000',
   className = '',
 }) => {
   return (
     <svg
       className={className}
       xmlns="http://www.w3.org/2000/svg"
-      width={size}
-      height={size}
+      width={24}
+      height={24}
       fill="none"
-      viewBox={`0 0 ${size} ${size}`}
+      viewBox={'0 0 24 24'}
```

Comment:
Noticed other parts of the codebase shifted from 24 magic number to variable but this one variable to magic number. Is this intentional?

Reply:
Yeah the SVG data itself only fits in a 24*24 viewbox

Reply:
I only changed CloseIcon to variable as we can manipulate the viewbox to create a smaller cross since it's symmetrical